install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -72,7 +72,7 @@ data:
       # and manipulation.
       # The "TPROXY" mode also configures the sidecar to run with the
       # CAP_NET_ADMIN capability, which is required to use TPROXY.
-      interceptionMode: REDIRECT
+      #interceptionMode: REDIRECT
```

Comment:
why this?

Reply:
It breaks people using the old istioctl and the upgrades. 

In general default values don't need to be defined explicitly.